src/Chainweb/Miner/Config.hs

```diff
@@ -45,24 +66,49 @@ makeLenses ''MinerConfig
 defaultMinerConfig :: MinerConfig
 defaultMinerConfig = MinerConfig
     { _configTestMiners = MinerCount 10
-    , _configMinerInfo = noMiner
+        -- hidden configuration option that is only used in testing
+    , _configMinerInfo = invalidMiner
+    , _configMinimumBlockHeight = 1200
     }
+  where
+    invalidMiner = Miner
```

Comment:
Shouldn't this be the same as `noMiner` (or `noMiner` should be renamed to `invalidMiner`) which is used elsewhere?

Reply:
`noMiner` the currently used for mining by test-nodes on devnet network and testnet. It's probably a poorly chosen name. `invalidMiner` here is just a configuration "template". Configuration validation detects it and reminds the user to fill in a value.